tests/ensemble_test.py

```diff
@@ -28,10 +33,14 @@ def test_import_before_deprecation():
         prior_mean = zeros(2)
         prior_cov = eye(2)
         prior = MultivariateNormal(loc=prior_mean, covariance_matrix=prior_cov)
-        simulator, prior = prepare_for_sbi(
+
+        prior, _, prior_returns_numpy = process_prior(prior)
```

Comment:
we actually don't need to process the prior and simulator here. it's just a test and we are defining a proper prior and simulator ourselves. just use the prior as it is and define the simulator via the lambda fun, or via a normal `def simulator(...)`. 
:)